install_config/storage_examples/gluster_dynamic_example.adoc

```diff
@@ -14,272 +15,35 @@ toc::[]
-kind: Pod
-metadata:
-  name: nginx-pod
-  labels:
-    name: nginx-pod
-spec:
-  containers:
-  - name: nginx-pod
-    image: gcr.io/google_containers/nginx-slim:0.8
-    ports:
-    - name: web
-      containerPort: 80
-    securityContext:
-      privileged: true
-    volumeMounts:
-    - name: gluster-vol1
-      mountPath: /usr/share/nginx/html
-  volumes:
-  - name: gluster-vol1
-    persistentVolumeClaim:
-      claimName: gluster1 <1>
-----
-<1> The name of the PVC created in the previous step.
-====
-
-. Create the Pod YAML file, save it, then submit it to {product-title}:
-+
-----
-$ oc create -f nginx-pod.yaml
-pod "gluster-pod1" created
-----
-
-. View the pod:
-+
-----
-$ oc get pods -o wide
-NAME                               READY     STATUS    RESTARTS   AGE       IP               NODE
-nginx-pod                          1/1       Running   0          9m        10.38.0.0        node1
-glusterfs-node0-2509304327-vpce1   1/1       Running   0          1d        192.168.10.100   node0
-glusterfs-node1-3290690057-hhq92   1/1       Running   0          1d        192.168.10.101   node1
-glusterfs-node2-4072075787-okzjv   1/1       Running   0          1d        192.168.10.102   node2
-heketi-3017632314-yyngh            1/1       Running   0          1d        10.42.0.0        node0
-----
-+
-[NOTE]
-====
-This may take a few minutes, as the the pod may need to download the image if it does not already exist.
-====
+include::install_config/persistent_storage/topics/glusterfs_security_selinux.adoc[]
 
-. `oc exec` into the container and create an *_index.html_* file in the
-`mountPath` definition of the pod:
-+
-----
-$ oc exec -ti nginx-pod /bin/sh
-$ cd /usr/share/nginx/html
-$ echo 'Hello World from GlusterFS!!!' > index.html
-$ ls
-index.html
-$ exit
-----
+[[dynamic-provisioning]]
+== Dynamic Provisioning
 
-. Using the `curl` command from the master node, `curl` the URL of the pod:
-+
-----
-$ curl http://10.38.0.0
-Hello World from GlusterFS!!!
-----
+include::install_config/persistent_storage/topics/glusterfs_dynamic_provisioning.adoc[]
 
-. Check your Gluster pod to ensure that the *_index.html_* file was written.
-Choose any of the Gluster pods:
-+
-----
-$ oc exec -ti glusterfs-node1-3290690057-hhq92 /bin/sh
-$ mount | grep heketi
-/dev/mapper/VolGroup00-LogVol00 on /var/lib/heketi type xfs (rw,relatime,seclabel,attr2,inode64,noquota)
-/dev/mapper/vg_f92e09091f6b20ab12b02a2513e4ed90-brick_1e730a5462c352835055018e1874e578 on /var/lib/heketi/mounts/vg_f92e09091f6b20ab12b02a2513e4ed90/brick_1e730a5462c352835055018e1874e578 type xfs (rw,noatime,seclabel,nouuid,attr2,inode64,logbsize=256k,sunit=512,swidth=512,noquota)
-/dev/mapper/vg_f92e09091f6b20ab12b02a2513e4ed90-brick_d8c06e606ff4cc29ccb9d018c73ee292 on /var/lib/heketi/mounts/vg_f92e09091f6b20ab12b02a2513e4ed90/brick_d8c06e606ff4cc29ccb9d018c73ee292 type xfs (rw,noatime,seclabel,nouuid,attr2,inode64,logbsize=256k,sunit=512,swidth=512,noquota)
+[[using-the-storage]]
```

Comment:
@jarrpa - this anchor tag is used in the gluster_example.adoc file as well. Can you change this to be more specific to this topic?